bodhi/tests/server/services/test_updates.py

```diff
@@ -3569,7 +3569,6 @@ def test_enabled_button_for_autopush(self, *args):
         resp = self.app.get(f"/updates/{resp.json['alias']}", headers={'Accept': 'text/html'})
         assert 'text/html' in resp.headers['Content-Type']
         assert nvr in resp
-        assert 'Stable by Karma' in resp
```

Comment:
Since `Stable by Karma` is now always displayed, we could use the tooltip text:
`assert 'The update will be automatically pushed to stable when karma reaches' in resp`